blog-posts/06_03_2021.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Home</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+
+<body>
+    <header>
+        <div class="navbar-container">
+            <h1 class="website-title">Sara Vega</h1>
+            <nav>
+                <!-- Main Navigation inside <header> -->
+                <a href="index.html">Home</a>
+                <a href="blog.html">Blog</a>
+                <a href="about.html">About me</a>
+                <a href="contact.html">Contact</a>
```

Comment:
These files are outside of the `blog-posts` folder that `06_23_2020.html` is in so the file path needs to reflect exiting out of the current directory to get to the parent directory where these files are located via `../filename.html`. 

```suggestion
                <a href="../index.html">Home</a>
                <a href="../blog.html">Blog</a>
                <a href="../about.html">About me</a>
                <a href="../contact.html">Contact</a>
```